styles/hackage/catppuccin.user.css

```diff
@@ -0,0 +1,574 @@
+    @yellow: @catppuccin[@@lookup][@yellow];
+    @green: @catppuccin[@@lookup][@green];
+    @teal: @catppuccin[@@lookup][@teal];
+    @sky: @catppuccin[@@lookup][@sky];
+    @sapphire: @catppuccin[@@lookup][@sapphire];
+    @blue: @catppuccin[@@lookup][@blue];
+    @lavender: @catppuccin[@@lookup][@lavender];
+    @text: @catppuccin[@@lookup][@text];
+    @subtext1: @catppuccin[@@lookup][@subtext1];
+    @subtext0: @catppuccin[@@lookup][@subtext0];
+    @overlay2: @catppuccin[@@lookup][@overlay2];
+    @overlay1: @catppuccin[@@lookup][@overlay1];
+    @overlay0: @catppuccin[@@lookup][@overlay0];
+    @surface2: @catppuccin[@@lookup][@surface2];
+    @surface1: @catppuccin[@@lookup][@surface1];
+    @surface0: @catppuccin[@@lookup][@surface0];
+    @base: @catppuccin[@@lookup][@base];
+    @mantle: @catppuccin[@@lookup][@mantle];
+    @crust: @catppuccin[@@lookup][@crust];
+    @accent-color: @catppuccin[@@lookup][@@accent];
+
+    :root {
+      color-scheme: if(@lookup = latte, light, dark);
+    }
+
+    html {
+      background-color: @base;
+    }
+
+    body {
+      background-color: @base;
+      color: @text;
+    }
+
+    a:hover {
+      background-color: inherit;
+    }
+
+    a[href]:link {
+      color: @accent-color;
+    }
+
+    a[href]:visited {
+      color: mix(@text, @accent-color, 75%);
+    }
+
+    a[href].def:link,
+    a[href].def:visited {
+      color: @subtext0;
+    }
+
+    a[href].def:hover {
+      color: mix(@text, @subtext0, 40%);
+    }
+
+    .caption,
+    h1,
+    h2,
+    h3,
+    h4,
+    h5,
+    h6,
+    summary {
+      color: @accent-color;
+      filter: none;
+    }
+
+    .collapser::before,
+    .expander::before,
+    .noexpander::before {
+      color: @accent-color;
+    }
+
+    pre {
+      background-color: @mantle;
+      border-color: @surface1;
+    }
+
+    blockquote {
+      background-color: @surface0;
+      border-color: @accent-color;
+    }
+
+    .src {
+      background-color: @mantle;
+    }
+
+    /*
+     * Headers
+     */
+
+    #page-header,
+    #package-header {
+      background-color: @accent-color;
+      color: @surface1;
+      border-color: shade(@accent-color, 25%);
+    }
+
+    #page-header .caption,
+    #package-header .caption,
+    #page-header a:link,
+    #page-header a:visited {
+      color: @base !important;
+    }
+
+    #module-header .caption {
+      color: @accent-color;
+      border-color: @surface1;
+    }
+
+    #module-header {
+      background-color: inherit;
+    }
+
+    #package-header #page-menu a:link,
+    #package-header #page-menu a:visited,
+    #package-header a:link,
+    #package-header a:visited {
+      color: @base;
+    }
+
+    #package-header a:hover {
+      background-color: transparent;
+    }
+
+    /*
+     * Header links
+     */
+
+    #package-header ul.links li {
+      border-color: @overlay0;
+    }
+
+    #package-header ul.links li:hover {
+      background-color: transparent;
+    }
```

Comment:
```suggestion
    /*
     * Header links
     */
    #package-header {
      &, #page-menu {
        a:link, a:visited {
          color: @base;
        }
      }
      
      a:hover {
        background-color: transparent;
      }
      
      ul.links li {
        border-color: @overlay0;
        
        &:hover {
          background-color: transparent;
        }
      }
    }
```

Reply:
Oops I missed L171-L175 as well, you don't need to use my suggestion exactly but see how that nesting can help reduce repetition and group similar elements together (without explicit comments necessarily)?